src/metrics/components/MetricsActions/MetricsActions.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { render, waitForI18n } from '../../../test-utils';
```

Comment:
Why not importing directly from the react testing library?

Reply:
@lavocatt , that also we can do. but during the test setup we created the test-utils file and added all the test related export in that file and whenever we need any import we just import the test-utils. files instead separately importing.
i.e (export * from '@testing-library/react';)